uptane-standard.md

```diff
@@ -240,28 +277,222 @@ The Time Server SHALL expose a public interface allowing primaries to communicat
+    4. Check that each targets metadata filename listed in the previous snapshot metadata file is also listed in this snapshot metadata file.
+    5. Check that the latest attested time is lower than the expiration timestamp in this metadata file. (Checks for a freeze attack.)
+10. Download and check the top-level targets metadata file from the image repository:
+    1. Check that it has been signed by the threshold of keys specified in the root metadata file.
+    2. Check that the version number of the previous targets metadata file, if any, is less than or equal to the version number of this targets metadata file. (Checks for a rollback attack.) {#test}
+    3. Check that the latest attested time is lower than the expiration timestamp in this metadata file. (Checks for a freeze attack.)
+    4. Check that the version number in this targets metadata file matches the version number given for it in the snapshot metadata file. (Checks for a mix-and-match attack.)
+11. For each image listed in the targets metadata file from the director repository, locate a targets metadata file that contains an image with exactly the same file name. For each delegated targets metadata file that is found to contain metadata for the image currently being processed, perform all of the checks in step 10. Use the following process to locate image metadata:
+    1. If the top-level targets metadata file contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
+    2. Recursively search the list of delegations, in order of appearance:
+        1. If it is a multi-role delegation, recursively visit each role, and check that each has signed exactly the same non-custom metadata (i.e., length and hashes) about the image. If it is all the same, return the metadata to be checked and skip to step 11.3.
+        2. If it is a terminating delegation and it contains signed metadata about the image, return the metadata to be checked and skip to step 11.3. If metadata about an image is not found in a terminating delegation, return an error code indicating that the image is missing.
+        3. Otherwise, continue processing the next delegation, if any. As soon as a delegation is found that contains signed metadata about the image, return the metadata to be checked and skip to step 11.3. 
+        4. If no signed metadata about the image can be found anywhere in the delegation tree, return an error code indicating that the image is missing.
+    3. Check that the targets metadata from the image repository matches the targets metadata from the director repository:
+        1. Check that the length and hash of the image are the same in both sets of metadata.
+        2. Check that the hardware identifier and release counter are the same in both sets of metadata.
+        3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
```

Comment:
Again with tolerating equality of the counter. Why?

Reply:
An OEM is allowed to use the same release counter for multiple versions if they want to upgrade / downgrade between them.